### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by antstorm**
**What changed? Why?**
Adds a template that will be used by default in all PRs. Following this will increase visibility into the changes that are being added to the project and should make it easier to find a culprit PR if there's an issue or get more context on what is going on. More info in #3087 

**How has it been tested?**
This PR message should prove that it works

**Potential risk level (on a scale from 1 to 5)**
1

### CODE REVIEW

**Comment by antstorm** [comment]
For some reason @claassistantio won't verify that I've already signed the agreement ^

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- Describe the future self what has changed and most importanly why -->
+**What changed? Why?**
```
Comment (yux0) [inline PULL_REQUEST_TEMPLATE.md]: Can we separate this to two questions?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+<!-- How have you verified this change? Tested locally? Added a unit test? Checked in staging env? -->
+**How has it been tested?**
+
+
+<!-- How risky is it to deploy this straight to production? -->
+**Potential risk level (on a scale from 1 to 5)**
```
Comment (yux0) [inline PULL_REQUEST_TEMPLATE.md]: Could it be more clear about the scale? 1 is the highest risk or 5 is the highest risk?

Comment (yux0) [reply PULL_REQUEST_TEMPLATE.md]: In addition, the number could be arbitrary if there is no criteria about risk level. What about changing the question to: What could be broken (What is the worst case) when deploying this change to production?

**Comment by yycptt** [comment]
> For some reason @claassistantio won't verify that I've already signed the agreement ^

Try close and reopen this pull request. It usually resolves the issue.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (antstorm) [reply PULL_REQUEST_TEMPLATE.md]: Good idea 👍

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (antstorm) [reply PULL_REQUEST_TEMPLATE.md]: I'll clarify this, thanks for pointing out!

**Comment by antstorm** [comment]
@yux0 I've added suggested changes

**Comment by antstorm** [comment]
@yux0 I've made suggested changes

**Comment by antstorm** [comment]
@yux0 thanks for approving, let me close it and re-open a new one to get through CLA

**Pull request closed without merging**